pkg/constants/constants.go

```diff
@@ -45,17 +45,17 @@ const (
 	// EnvoyMetricsCluster is the cluster name of the Prometheus metrics cluster
 	EnvoyMetricsCluster = "envoy-metrics-cluster"
 
-	// EnvoyZipkinCluster is the name of the Zipkin cluster.
-	EnvoyZipkinCluster = "envoy-zipkin-cluster"
+	// EnvoyTracingCluster is the default name to refer to the tracing cluster.
+	EnvoyTracingCluster = "envoy-tracing-cluster"
 
-	// DefaultZipkinEndpoint is the Zipkin endpoint.
-	DefaultZipkinEndpoint = "/api/v2/spans"
+	// DefaultTracingEndpoint is the default endpoint route.
```

Comment:
What does `default endpoint route` imply here?

Reply:
I had to actually check, it's the default http path where where the metrics are pushed via POST
https://zipkin.io/zipkin-api/#/default/post_spans

Could be hardcoded maybe? it's part of the zipkin v2 api and this should not change I think

Reply:
My question was more to do with the term `endpoint route`, which seems a bit off in the context of tracing.

Reply:
`http path`? `http api path`?